blog/2023-11-20-stately-studio-2-0/index.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Stately Studio 2.0'
+description: We’re so excited to release Stately Studio 2.0, which aims to meet developers where they are, no matter which libraries, frameworks, or even languages they use.
+authors: [david]
+tags: [new, Stately, AI, Sky, XState, Redux, Zustand, generate, React ]
+date: 2023-11-20
+slug: 2023-11-20-stately-studio-2-0
+image: /blog/2023-11-20-stately-studio-2-0.png
+---
+
+It’s been about a year since we’ve released [Stately Studio 1.0](https://stately.ai/blog/2022-10-18-introducing-stately-studio-1-0), and a lot has happened. [Stately Studio](https://stately.ai/editor) is essentially a visual software modeling tool that strives to make it easy to create, manage, and use state machines, no matter how complex they may get. Primarily, it served as a powerful set of devtools for [XState](https://github.com/statelyai/xstate) (an open-source library for creating state machines, statecharts and actors in JavaScript and TypeScript). You could import XState code to a state diagram, modify it visually in an intuitive drag-and-drop canvas, and export to XState. Eventually, we added more export options: JSON, Markdown, Mermaid diagrams, and stories.
+
+But Stately Studio has bigger ambitions than just being a suite of devtools for XState. We’ve frequently heard that these state diagrams are an important **source of truth** for critical app logic, serving as documentation for the entire team that stays up-to-date with your code. But a reliable source of truth for app logic is a need for **all** apps, not just those that use state machines directly.
+
+That’s why we’re so excited to release Stately Studio 2.0, which aims to **meet developers where they are**, no matter which libraries, frameworks, or even languages they use. There are many benefits to modeling app logic with state diagrams and the actor model, and we want to enable developers to take advantage of those benefits to build more robust, feature-rich, and maintainable app logic faster.
+
+<!--truncate-->
+
+[image]
+
+Stately Studio 2.0 is an intelligent software modeling platform for working with app logic at all stages of the software development lifecycle. With its many specialized graph algorithms for analyzing state machines deterministically, paired with AI (large language models (LLMs) and more) for the not-so-deterministic tasks, such as generation & modification from natural language, [Stately Studio](https://stately.ai) is an essential tool for any development team working on complex app logic.
```

Comment:
Really agree with this suggestion @gavination. I would add that the "and more" is awkward between the "(large language models)" and "to handle..." and doesn't add anything. It can just be removed.

Another option is to end with "Stately Studio is now and essential tool", to emphasize the "arrival" with this release, but if others disagree then I don't feel strongly about it.

Reply:
Also, is there a reason to use the ampersand in "generation & modification" in this one location here as opposed to just "and"?